templates/broker-auth.clusterrole.yaml

```diff
@@ -0,0 +1,42 @@
+---
+
+#apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: v1
+kind: ClusterRole
+metadata:
+  name: '{{ broker_name }}-auth'
+rules:
+  - apiGroups: [""]
+    resources: ["namespaces"]
+    verbs: ["create", "delete"]
+  - apiGroups: ["authorization.k8s.io"]
+    resources: ["subjectaccessreviews"]
+    verbs: ["create"]
+  - apiGroups: ["authentication.k8s.io"]
+    resources: ["tokenreviews"]
+    verbs: ["create"]
+  - apiGroups: ["networking.k8s.io", ""]
+    # attributeRestrictions: null
+    resources: ["networkpolicies"]
+    verbs: ["create", "delete"]
+  - apiGroups: ["automationbroker.io"]
+    # attributeRestrictions: null
```

Comment:
These aren't actually supported in the python-client. What happens if we don't specifically set them to `null`?

Reply:
I don't think anything happens